.github/workflows/regression_tests.yml

```diff
@@ -26,6 +26,14 @@ jobs:
           python -m pip install --upgrade pip
           python -m pip install -r requirements.txt
           pip freeze | egrep 'iree|tensorflow'
+          python -m pip install flake8 pytest
+          
+      - name: Lint with flake8
+        run: |
+          # stop the build if there are Python syntax errors or undefined names
```

Comment:
Is the repo actually lint clean? We shouldn't add new blocking lint checks if they aren't yet green.